gnuradio-runtime/include/gnuradio/tags.h

```diff
@@ -33,15 +33,15 @@ struct GR_RUNTIME_API tag_t {
     //! ignore this.
     std::vector<long> marked_deleted;
 
-    /*!
-     * Comparison function to test which tag, \p x or \p y, came
-     * first in time
-     */
-    static inline bool offset_compare(const tag_t& x, const tag_t& y)
+    //! Comparison function to test which tag, \p x or \p y, came first in time
+    friend inline bool operator<(const tag_t& x, const tag_t& y)
     {
         return x.offset < y.offset;
     }
+    //! Comparison function to test which tag, \p x or \p y, came first in time
+    static inline bool offset_compare(const tag_t& x, const tag_t& y) { return x < y; }
```

Comment:
Since this file is in the public API, should the implementations be moved into `tag.cc`?

Reply:
ah wait, can't be inline then. So, probably not.

Reply:
doh - yep merging now.